### DISCUSSION TITLE
React native Android Wearable

### DISCUSSION THREAD

**Discussion by HadiKhalifeh** [react-native-community/discussions-and-proposals#465] [Ideas]
Hello,
How to establish a connection between react native and watchOS..
AppleWatch:we could do that using react-native-watch-connectivity
how about android wearables(no library or a clear docummentation)
Thank You

**Comment by muhammadqazi** [comment]
I want to build react native app for watch os but still nothing about it.

**Comment by Carlene12** [comment]
To connect your React Native App with your WatchOS App you can send a message from the iPhone[.](https://rzstatus.com/desi-status-in-punjabi/) Simulator to the apple watch simulator.

**Comment by fabOnReact** [comment]
You can easily connect mobile and wear devices using https://github.com/fabOnReact/react-native-wear-connectivity
